packages/core/admin/admin/src/content-manager/pages/EditView/components/Header.tsx

```diff
@@ -1,264 +1,52 @@
-import * as React from 'react';
-
-import {
-  Box,
-  Button,
-  Dialog,
-  DialogBody,
-  DialogFooter,
-  Flex,
-  HeaderLayout,
-  Typography,
-} from '@strapi/design-system';
+import { Flex, Status, Typography } from '@strapi/design-system';
 import { Link } from '@strapi/design-system/v2';
-import { AllowedActions, useCMEditViewDataManager } from '@strapi/helper-plugin';
-import { ArrowLeft, Check, ExclamationMarkCircle } from '@strapi/icons';
-import get from 'lodash/get';
-import isEqual from 'lodash/isEqual';
+import { ArrowLeft } from '@strapi/icons';
 import { useIntl } from 'react-intl';
-import { NavLink, useNavigate } from 'react-router-dom';
 
+import { capitalise } from '../../../../utils/strings';
 import { getTranslation } from '../../../utils/translations';
 
 interface HeaderProps {
-  allowedActions: AllowedActions;
+  isCreating?: boolean;
+  status?: 'draft' | 'published' | 'modified';
 }
 
-const Header = ({ allowedActions: { canUpdate, canCreate, canPublish } }: HeaderProps) => {
-  const {
-    initialData,
-    isCreatingEntry,
-    isSingleType,
-    status,
-    layout,
-    modifiedData,
-    onPublish,
-    onUnpublish,
-    publishConfirmation: { show: showPublishConfirmation, draftCount } = {},
-    onPublishPromptDismissal,
-  } = useCMEditViewDataManager();
-  const navigate = useNavigate();
-  const [showWarningUnpublish, setWarningUnpublish] = React.useState(false);
+const Header = ({ isCreating, status = 'draft' }: HeaderProps) => {
   const { formatMessage } = useIntl();
 
-  const currentContentTypeMainField = get(layout, ['settings', 'mainField'], 'id');
-  const currentContentTypeName = get(layout, ['info', 'displayName'], 'NOT FOUND');
-  const didChangeData =
-    !isEqual(initialData, modifiedData) ||
-    (isCreatingEntry && Object.keys(modifiedData).length > 0);
-
-  const createEntryIntlTitle = formatMessage({
-    id: getTranslation('containers.Edit.pluginHeader.title.new'),
-    defaultMessage: 'Create an entry',
-  });
-
-  let title = createEntryIntlTitle;
-
-  if (!isCreatingEntry && !isSingleType) {
-    title = initialData[currentContentTypeMainField] || currentContentTypeName;
-  }
-
-  if (isSingleType) {
-    title = currentContentTypeName;
-  }
-
-  let primaryAction = null;
-
-  if (isCreatingEntry && canCreate) {
-    primaryAction = (
-      <Flex gap={2}>
-        <Button disabled startIcon={<Check />} variant="secondary">
-          {formatMessage({ id: 'app.utils.publish', defaultMessage: 'Publish' })}
-        </Button>
-        <Button disabled={!didChangeData} loading={status === 'submit-pending'} type="submit">
-          {formatMessage({
-            id: getTranslation('containers.Edit.submit'),
-            defaultMessage: 'Save',
-          })}
-        </Button>
-      </Flex>
-    );
-  }
+  const title = isCreating
```

Comment:
Why would we want a boolean as a title

Reply:
It's not, the lines below are a ternary 👍🏼